tools/dbus_ignorelist

```diff
@@ -1660,6 +1660,7 @@ org.freedesktop.systemd1.Socket.OOMScoreAdjust
 org.freedesktop.systemd1.Socket.PAMName
 org.freedesktop.systemd1.Socket.PassCredentials
 org.freedesktop.systemd1.Socket.PassEnvironment
+org.freedesktop.systemd1.Socket.PassFileDescriptorsToExec
```

Comment:
No. This is for methods/properties that don't need explicit version info, i.e. added before v183. Please add proper version info for new properties.

Reply:
Oh, okay. Let me address that.